README.md

```diff
@@ -60,6 +68,7 @@ UserApp.get_all_users = function() {
   })
   .done(function(data) {
     console.log("success!")
+  UserApp.get_all_users(data);  // was the function not being called to display the data?
```

Comment:
No, the biggest problem is that `UserApp.add_all_users(data);` needs to be INSIDE of the `.done()` callback in order to have access to `data`.